activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -484,6 +486,97 @@ def test_only_check_for_insensitive_comparison_capability_once
         end
       end
 
+      def test_ignores_warnings_when_behaviour_ignore
+        old_action = ActiveRecord.db_warnings_action
+        ActiveRecord.db_warnings_action = :ignore
+        @connection.execute("do $$ BEGIN RAISE WARNING 'foo'; END; $$")
```

Comment:
This test should also assert something. Or we should remove it if there's nothing to assert.